setup.py

```diff
@@ -215,6 +215,9 @@ def read(*rnames):
 #   }
 }
 
+if 'RADICAL_DEBUG' in os.environ:
+    setup_args['install_requires'].append('setproctitle')
+
```

Comment:
Yeah, so that logic is broken. Everything that is not explicitly in the bootstrapper ends up in rp_install instead of the ve.
I'll open a branch for that.